ui/src/components/Liquidity/Liquidity.jsx

```diff
@@ -108,20 +126,44 @@ const Liquidity = () => {
                 All liquidity pairs currently use {centralInfo?.petname}
               </h2>
             </div>
-            <Tab.Group>
-              <Tab.List className="bg-white  text-md  rounded-sm  flex ">
-                <Tab className={tabClasses}>Add</Tab>
-                <Tab className={tabClasses}>Remove</Tab>
-              </Tab.List>
-              <Tab.Panels>
-                <Tab.Panel>
-                  <AddLiquidity />
-                </Tab.Panel>
-                <Tab.Panel>
-                  <RemoveLiquidity setOpen={setOpen} />
-                </Tab.Panel>
-              </Tab.Panels>
-            </Tab.Group>
+            <div>
+              <div className="bg-white text-md rounded-sm flex mb-4">
+                <div
+                  className={tabClass(0)}
+                  onClick={() => {
+                    setTabIndex(0);
+                  }}
+                >
+                  Add
+                </div>
+                <div
+                  className={tabClass(1)}
+                  onClick={() => {
+                    setTabIndex(1);
+                  }}
+                >
+                  Remove
+                </div>
+              </div>
+              {tabIndex === 0 && (
+                <motion.div>
+                  {assetloader ? (
+                    <CustomLoader text="Loading Assets..." size={25} />
+                  ) : (
+                    <AddLiquidity />
```

Comment:
Add liquidity should not show the central asset since all liquidity pairs are with the central asset and it does not make sense to have a liquidity pair with it. @Hussain-7 
There's a filter that filters out the central asset, something is broken there, please fix. It's somewhere in assetDialog or something, find the appropriate dialog and you shall find the filter somewhere there.